modules/shapefile/docs/api-reference/dbf-loader.md

```diff
@@ -0,0 +1,47 @@
+# DBFLoader
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.3-blue.svg?style=flat-square" alt="From-v2.3" />
+</p>
+
+Shapefile loader
+
+| Loader                | Characteristic                               |
+| --------------------- | -------------------------------------------- |
+| File Extension        | `.shp`,                                      |
+| File Type             | Binary                                       |
+| File Format           | Shapefiles                                   |
+| Data Format           | [Table](/docs/specifications/category-table) |
+| Supported APIs        | `load`, `parse`, `parseSync`                 |
+| Decoder Type          | Synchronous                                  |
+| Worker Thread Support | Yes                                          |
+
+## Usage
+
+```js
+import {DBFLoader} from '@loaders.gl/shapefile';
+import {load} from '@loaders.gl/core';
+
+const data = await load(url, DBFLoader);
+```
+
+## Options
+
+N/A
+
+## Format Summary
+
+ESRI Shapefiles are the historical standard for storing Geospatial vector data.
```

Comment:
This paragraph should be reworded. 

"the historical" is a strong word. Sounds like a museum...

"The format has been around since the late 90s, so a huge amount of data is in
this format." - strange sentence to have here.

Reply:
I'm not always great at writing format summaries :) this is copied from `shp-loader.md`, so it should be updated in both places.